packages/shared/src/normalizeProp.ts

```diff
@@ -28,7 +28,7 @@ export function normalizeStyle(
 }
 
 const listDelimiterRE = /;(?![^(]*\))/g
-const propertyDelimiterRE = /:(.+)/
+const propertyDelimiterRE = /:(.+)/s
```

Comment:
dotAll flag is a ES2018 feature so it will not work in certain browsers
```suggestion
const propertyDelimiterRE = /:([^]+)/
```